web-client/src/presenter/actions/TrialSession/computeTrialSessionFormDataAction.ts

```diff
@@ -28,6 +28,9 @@ export const computeTermAndUpdateState = (
 
     store.set(state.form.term, term);
     store.set(state.form.termYear, date.year);
+  } else {
+    store.set(state.form.term, undefined);
+    store.set(state.form.termYear, undefined);
```

Comment:
Is this necessary? (I don't know--just double-checking!) If so, I'd recommend using `store.unset`, e.g., `state.unset(state.form.term)`. (It does the same thing, just a little clearer and easier to search for.) But it's not a biggie.